test/e2e/network/service.go

```diff
@@ -57,10 +57,10 @@ var _ = SIGDescribe("Services", func() {
 		if CurrentGinkgoTestDescription().Failed {
 			framework.DescribeSvc(f.Namespace.Name)
 		}
-		for _, lb := range serviceLBNames {
-			framework.Logf("cleaning gce resource for %s", lb)
-			framework.CleanupServiceGCEResources(cs, lb, framework.TestContext.CloudConfig.Zone)
-		}
+		//for _, lb := range serviceLBNames {
```

Comment:
remove dead code.

Reply:
And, why remove these lines?

Reply:
oops, that's a mistake, I did this for local test.